### PR TITLE
Added fallback for IE9 in treeColor

### PR DESCRIPTION
**Pull request by weskamm**
Fixes a minor coloring issue in IE9

### CODE REVIEW

File: src/view/panel/LegendTree.js [hunk 1]
```diff
@@ -202,6 +202,7 @@ Ext.define("BasiGX.view.panel.LegendTree", {
                 '.' + elemenIdAndCssClass + ' { ' +
                 ' background-color: ' + color + ';' +
                 ' background:linear-gradient(to right, white, ' + color + ');' +
+                ' filter: alpha(opacity=50)\10;' +//IE9 fallback
```
Comment (marcjansen) [inline src/view/panel/LegendTree.js]: What does `\10` do?

**Comment by marcjansen** [comment]
I don't understand the syntax used, @weskamm. Can you answer my question?

**Comment by weskamm** [comment]
Its based on this advice and has been tested in IE9 - works:

http://stackoverflow.com/questions/11829792/ie-9-support-both-filter-and-rgba

**Comment by marcjansen** [comment]
Thanks, merge at will.

**Pull request merged**